Formula/ntfy.rb

```diff
@@ -0,0 +1,43 @@
+class Ntfy < Formula
+  desc "Send push notifications to your phone or desktop via PUT/POST"
+  homepage "https://ntfy.sh/"
+  url "https://github.com/binwiederhier/ntfy.git", tag: "v2.3.1", revision: "a75fb08ef138d67599499c1f78628a0b35fcef54"
+  license any_of: ["Apache-2.0", "GPL-2.0-only"]
+  head "https://github.com/binwiederhier/ntfy.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "cli-deps-static-sites"
+    ldflags = %W[
+      -X main.version=#{version}
+      -X main.date=#{time.strftime("%F")}
+      -X main.commit=#{Utils.git_head}
+      -s
+      -w
+    ]
+    with_env(
+      "CGO_ENABLED" => "0",
+    ) do
+      system "go", "build", *std_go_args(ldflags: ldflags), "-tags", "noserver"
+    end
+  end
+
+  test do
+    require "securerandom"
+    random_topic = SecureRandom.hex(6)
+    begin
+      ntfy_in = shell_output("#{bin}/ntfy publish #{random_topic} 'Test message from HomeBrew during build'")
+      ohai ntfy_in
+      sleep 1
+      ntfy_out = shell_output("#{bin}/ntfy subscribe --poll #{random_topic}")
+      ohai ntfy_out
+      assert_match ntfy_in, ntfy_out
+    end
```

Comment:
```suggestion

    ntfy_in = shell_output("#{bin}/ntfy publish #{random_topic} 'Test message from HomeBrew during build'")
    ohai ntfy_in
    sleep 1
    ntfy_out = shell_output("#{bin}/ntfy subscribe --poll #{random_topic}")
    ohai ntfy_out
    assert_match ntfy_in, ntfy_out
```
You don't need the `begin` ... `end` block here.

Reply:
Thanks for the pointer, fixed!